components/console-backend-service/internal/domain/k8s/limitrange_service.go

```diff
@@ -33,3 +39,54 @@ func (svc *limitRangeService) List(ns string) ([]*v1.LimitRange, error) {
 
 	return result, nil
 }
+
+func (svc *limitRangeService) Create(namespace string, name string, limit gqlschema.LimitRangeInput) (*v1.LimitRange, error) {
+	parsedMaxMemory, errMaxMemory := resource.ParseQuantity(*limit.Max.Memory)
+	if errMaxMemory != nil {
+		return nil, errors.Wrap(errMaxMemory, "Error while creating limit range, max.memory")
```

Comment:
``` go
return nil, errors.Wrapf(errMaxMemory, "Error while creating %s, max.memory", pretty.LimitRange)
```

Reply:
You may want to aggregate errors as @akucharska did in [resourcequota_service.go#L56](https://github.com/kyma-project/kyma/blob/master/components/console-backend-service/internal/domain/k8s/resourcequota_service.go#L56)

Reply:
done